source/solvers/mf_navier_stokes_operators.cc

```diff
@@ -691,22 +711,58 @@ NavierStokesSUPGPSPGOperator<dim, number>::do_cell_integral_local(
             }
         }
 
+      if (this->stabilization ==
+          Parameters::Stabilization::NavierStokesStabilization::gls)
```

Comment:
Can't you make this a template parametr of the operator so that these if become if constexpr? I'm afraid that at one point that will make up for a lot of ifs...